lang/csharp/build.sh

```diff
@@ -30,7 +30,11 @@ case "$1" in
 
   test)
     xbuild
-    nunit-console Avro.nunit
+    if [[ ${INSIDE_AVRO_DOCKER} == 'Yes' ]]; then
+      nunit-console --framework=4.0 Avro.nunit
```

Comment:
Any reason why there is a difference in the two?
The reasoning behind my question is that doing a split will mean that the docker tests will be different from the normal tests.  Is it possible to add the framework and console directly without the if check?